app/src/main/res/layout/fragment_navigation_overlay_top_nav.xml

```diff
@@ -48,6 +48,16 @@
         android:layout_height="0dp"
         android:layout_weight="1" />
 
+    <!-- This button is set visible if the user is in the experiment. -->
+    <!-- TODO: add content description dynamically -->
+    <!-- TODO: use production icon. -->
+    <ImageButton
+        android:id="@+id/fxaButton"
+        style="@style/NavigationButton"
+        android:src="@drawable/lb_ic_cc"
+        android:contentDescription="@null"
+        android:visibility="gone"/>
```

Comment:
Up to you, but I would consider wrapping this in a `KillSwitchLayout` (for paranoia reasons).

Reply:
That makes sense to me and it's good to be consistent. Thanks for calling this out.

Reply:
I tried wrapping this view and it doesn't just work so, instead of spending 30min debugging, I'm just going to leave it as is. I'm not concerned because this view's use case is trivial – it's on or it's off – and doesn't manage complex visibility state with animations like our previous experiment.

We can use this as a learning experiment, if we run into problems because of this later, we can be even more careful to use KillswitchLayout.